examples/dataservice-read/example.cpp

```diff
@@ -73,15 +73,27 @@ int main() {
           catalogResponse.GetResult();
       LOG_INFO_F("read-example", "Catalog description: %s",
                  responseResult.GetDescription().c_str());
+      auto layers = catalogResponse.GetResult().GetLayers();
+      if (!layers.empty()) {
+        firstPartitionId = layers.front().GetId();
+      }
+      auto end = layers.size() <= gMaxLayers ? layers.end()
+                                             : layers.begin() + gMaxLayers;
+      for (auto it = layers.cbegin(); it != end; ++it) {
+        LOG_INFO_F("read-example", "Layer '%s' (%s): %s", it->GetId().c_str(),
```

Comment:
LOG_INFO_F(**"read-example"**, "Layer '%s' (%s): %s", it->GetId().c_str(),

Move `read-example` string as `constexpr auto` into the anonymous namespace (suggested name of variable - `LOG_TAG`).

Reply:
done

Reply:
Better to rename this variable as a `gLogTag` instead of `gLogLabel`, because first parameter of LOG_... is actually named `tag`:
`#define LOG_INFO_F(tag, ...) \
  LOG_F(::olp::logging::Level::Info, tag, __VA_ARGS__)`